microsoft-edge/webview2/release-notes.md

```diff
@@ -27,6 +27,71 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 
+<!-- ## 1.0.1010-prerelease
+
+Release Date: September 13, 2021  
+
+[NuGet package for WebView2 SDK 1.0.955-prerelease][NuGetGallery1.0.955-prerelease]
+
+For full API compatibility, this prerelease version of the WebView2 SDK requires Microsoft Edge version 94.0.1010.0 or higher.
+
+### General  
+*   General WebView2 performance improvements.
+*   General reliability fixes. \([\#1605][GithubMicrosoftedgeWebviewfeedbackIssue1605] and[\#1678][GithubMicrosoftedgeWebviewfeedbackIssue1678]\)
+*   Added launch and foreground boost support.
+
+#### Experimental Features  
+*   `EnsureCoreWebView2Async` will now throw an `ArgumentException` if called multiple times with incompatible parameters, such as new environments, to avoid silent failures.
+*   Added [UserDataFolder][Webview2ReferenceWin32Icorewebview2experimentalenvironment5ViewWebview210999PrereleaseGetUserdatafolder] property to Environment Interface. 
```

Comment:
We're changing the default handling for the user data folder. Maybe we should add something more descriptive and link to this announcement: https://github.com/MicrosoftEdge/WebView2Announcements/issues/26